src/Orcid/Provider.php

```diff
@@ -149,8 +149,7 @@ protected function getUserByToken($token)
                         $userUrl,
                         [RequestOptions::HEADERS => ['Content-Type' => 'application/vnd.orcid+xml',
                             'Accept'                                => 'application/json',
-                            'Authorization type'                    => 'Bearer',
-                            'Access token'                          => $token, ],
+                            'Authorization'                         => 'Bearer ' . $token,],
```

Comment:
```suggestion
                            'Authorization'                         => 'Bearer '.$token],
```

Reply:
or use `sprintf`

Reply:
Hi, the change without the additional whitespaces and comma is fine of course, so I think I can resolve this communication.